### PR TITLE
AWS Document Client refactor

### PR DESCRIPTION
**Pull request by Ludikrous**
Changes all AWS api objects from the regular version to Document Client, to remove the type notation from all endpoints.
All endpoints and tests have been updated!

All tests have passed with 99.99% code coverage.

### CODE REVIEW

File: backend/users/__tests__/invite_user.test.js [hunk 1]
```diff
@@ -39,11 +39,11 @@ describe('invite_user', () => {
   });
   
   it('Valid case', async () => {
-    console.log("WARNING: SENDGRID_API_KEY must be exposed as an env var!")
+    console.warn("WARNING: SENDGRID_API_KEY must be exposed as an env var!")
```
Comment (burbelohugo) [inline backend/users/__tests__/invite_user.test.js:42]: Can we get rid of this now that we pull it from AWS Secrets Manager?

**Comment by burbelohugo** [comment]
Additionally, this is going to break our request structure on the frontend. Can you edit the `general.js` mixin on the frontend so it uses this new request structure (aka doesn't need to deal with the type annotations)?

**Comment by Ludikrous** [comment]
> Additionally, this is going to break our request structure on the frontend. Can you edit the `general.js` mixin on the frontend so it uses this new request structure (aka doesn't need to deal with the type annotations)?

Looks like this was actually fixed in one of my bulk find/replace operations

**Comment by Ludikrous** [comment]
The test that failed was intermittent, re-ran the workflow and everything passed

**Comment by burbelohugo** [comment]
Users service tests failed due to a sendgrid outage, merging anyway since it's unrelated

**Pull request merged**